src/go/cmd/http-relay-server/server/server.go

```diff
@@ -377,6 +377,11 @@ func (s *Server) userClientRequest(w http.ResponseWriter, r *http.Request) {
 	// from the relay-client.
 	backendRespChan, err := s.relayRequest(ctx, *backendCtx, backendReq)
 	if err != nil {
+		if _, ok := err.(*RelayClientUnavailableError); ok {
+			w.WriteHeader(http.StatusServiceUnavailable)
+			w.Header().Set("X-CLOUDROBOTICS-HTTP-RELAY", backendCtx.Id)
+			return
```

Comment:
Please also return an error body here because people can see this error in their browser (and I think also in inctl output etc). `err.Error()` should be fine with the suggestion above.

Reply:
done